roles/deployment/definition/defaults/main.yml

```diff
@@ -99,6 +99,20 @@ database_defaults:
     name: schemaregistry
     user: schemaregistry
     password: "{{ database_default_password }}"
+  SQL_STREAM_BUILDER:
+    host: "{{ database_host }}"
+    port: "{{ database_type | cloudera.cluster.default_database_port }}"
+    type: "{{ database_type }}"
+    name: ssb_admin
+    user: ssb_admin
+    password: "{{ database_default_password }}"
+  SQL_STREAM_BUILDER_MVE:
```

Comment:
Is any magic required to make this DB get provisioned? I had a feeling that this is a list of services rather than a list of databases.

Reply:
This is a list of databases. There was no additional magic required. Just added the db here and it got created correctly.

Reply:
I'm surprised because here: https://github.com/cloudera-labs/cloudera.cluster/blob/main/roles/deployment/databases/tasks/mariadb.yml#L17 it's iterating through the databases using the list of services as a key. I'm struggling tos ee how SQL_STREAM_BUILDER_MVE would ever get created.

Reply:
Mmm... that's a good point, @tmgstevens . Let me review/retest this.

Reply:
@tmgstevens I've reviewed this and I believe there's no need for the MariaDB and PostgreSQL db creation role to filter the list of databases by the service names (`{{ databases | intersect(services) }}`). Removing that subtraction appears to be harmless to me and makes it more flexible, since you can specify multiple databases per service if needed, which is the case of SSB.

I've made the change and re-tested it and the database is now being created successfully and I haven't seen any collateral effects so far.